spring-pulsar/src/test/java/org/springframework/pulsar/core/ConsumerAcknowledgmentTests.java

```diff
@@ -361,4 +361,64 @@ void testBatchNackForEntireBatchWhenUsingBatchListener() throws Exception {
 		pulsarClient.close();
 	}
 
+	// *********************
```

Comment:
I'm not sure what the significance of this divider bar is, but do we need it in here?

Reply:
lol - user error.